Project.toml

```diff
@@ -0,0 +1,27 @@
+name = "Mux"
+uuid = "a975b10e-0019-58db-a62f-e48ff68538c9"
+version = "0.7.1"
+
+[deps]
+AssetRegistry = "bf4720bc-e11a-5d0c-854e-bdca1663c893"
+Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
+HTTP = "cd3eb016-35fb-5094-929b-558a96fad6f3"
+Hiccup = "9fb69e20-1954-56bb-a84f-559cc56a8ff7"
+Lazy = "50d2b5c4-7a5e-59d5-8109-a42b560f39c0"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
+WebSockets = "104b5d7c-a370-577a-8038-80a2059c5097"
+
+[compat]
+AssetRegistry = "0.1"
```

Comment:
Can these dependencies be opened up a bit more?

Reply:
Sure, although technically only WebSockets can be changed from `1.5` to `1` according to Semver if we want to guarantee non-breakage (because minor releases can have breaking changes while our deps are `<1.x`).

Reply:
Oh I meant something like:

`HTTP = "0.6, 0.7, 0.8` 

(Assuming those versions work with the project as well)